### DISCUSSION TITLE
Add "Locate" Button near PoseStamped topic in topic selector of 3D Panel

### DISCUSSION THREAD

**Discussion by sionyx** [foxglove/community#222] [Q&A]
Sometimes object position is relatively far from frame center so it hard to find where it is.

It would be nice to add Locate button near topics with type PoseStamped pressing on which will jump camera in current frame to coordinate specified by topic's pose.

See sample drawing below
![image](https://user-images.githubusercontent.com/1351998/149365236-3982cd4d-a4f1-4543-9a76-095cd2da795b.png)

**Comment by jtbandes** [comment]
Thanks for the request. I think this is related to the idea of being able to "follow" a marker or topic, rather than just a TF frame (since when you initiate follow mode we usually snap the followed frame to the center of the viewport). cc @jhurliman @defunctzombie
